utils/misc.go

```diff
@@ -67,7 +68,7 @@ func RandomID(randomIDLenth int) string {
 		randomIDLenth = types.RandomIDDefaultLength
 	}
 
-	uuid := UUID()
+	uuid := strings.Replace(UUID(), "-", "", -1)
```

Comment:
Consider covering this condition check in the [unit test](https://github.com/longhorn/go-common-libs/blob/main/utils/misc_test.go#L159-L160).

Reply:
Added. Thank you.